test/Intl/NumberFormat.js

```diff
@@ -4,7 +4,6 @@
 //-------------------------------------------------------------------------------------------------------
 
 // WinTH: Changed to use non-breaking space in some output
```

Comment:
I think you can remove this line at this point as well. It has varied a bit between certain releases, the regex covers it now.